salt/modules/win_timezone.py

```diff
@@ -9,6 +9,12 @@
 import logging
 import re
 
+try:
+    import tzlocal
```

Comment:
we seem to align on doing this like:
try:
import tzlocal
except ImportError:
tzlocal = None

And then instead of HAS_TZLOCAL just do - if tzlocal is not None:

Reply:
i see many instances in the code of using `HAS_*` in the same manner. Do you see an advantage to using None instead?

Reply:
well it seems to get rid of the pylint error. @rallytime do we have a standard for this? should i be using what @rares-pop recommended or just adding a pylint exception: `# pylint: disable=import-error`

Reply:
I had to impression that this is the new way of doing it.
I have seen @isbm recommending that in some of the PRs he reviewed.

The only advantage I've seen is one less variable. Maybe @isbm can give his insight on this?

Reply:
We do not have a standard way of doing it, and I am fine with either way. It's really just a preference at this point.